<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="baudrateLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusTextBrowser">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
